mobile/components/FormCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { useState } from "react";
+import { View, styled } from "tamagui";
+import Badge, { BadgeProps } from "./Badge";
+import Card from "./Card";
+import { Typography } from "./Typography";
+import CardFooter from "./CardFooter";
+
+export interface FormCardProps {
+  /**
+   * Header text
+   */
+  header: string;
+
+  /**
+   * Subheader optional text
+   */
+  subHeader?: string;
+
+  /**
+   * Footer text
+   */
+  footer: string;
+
+  /**
+   * Optional preset type.
+   */
+  badgeProps: BadgeProps;
+
+  /**
+   * Performed action for onPress
+   */
+  action: () => void;
+}
+
+const FormCard = (props: FormCardProps): JSX.Element => {
+  const { header, subHeader, footer, badgeProps, action } = props;
+  const hasSubHeader = subHeader ? subHeader.trim() !== "" : false;
+
+  const badgePreset = badgeProps.preset || "default";
+  const badgeChildren = badgeProps.children;
+
+  const CardHeader = styled(View, {
+    name: "CardHeader",
+    justifyContent: "space-between",
+    flexDirection: "row",
+    alignItems: "center",
+    marginBottom: 8,
```

Comment:
we have a 'xxs' token for this margin value

Reply:
Done